### PR TITLE
Fix headline font-size in standard story-promo

### PR DESCRIPTION
**Pull request by ghost**
Resolves #NUMBER

**Overall change:** Apply `getGreatPrimer` on standard story promo above 600px

**Code changes:**

- Update snapshots
- Use `getGreatPrimer` on standard story promo above 600px.
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-story-promo/src/index.jsx [hunk 1]
```diff
@@ -145,15 +146,17 @@ export const Headline = styled.h3`
...
-
     return topStory ? getParagon(script) : getPica(script);
   }}
   color: ${C_EBON};
   ${({ service }) => getSerifMedium(service)}
   margin: 0; /* Reset */
   padding-bottom: ${GEL_SPACING};
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    ${props => (props.script ? getParagon(props.script) : '')};
+    ${props =>
```
Comment (jroebu14) [inline packages/components/psammead-story-promo/src/index.jsx]: I think when dealing with props it's preferable to destructure e.g. `({ script, topStory }) =>`

**Review by jroebu14 — APPROVED** [review]
LGTM 👍

File: packages/components/psammead-story-promo/src/index.jsx (hunk 1 above)
Comment (ghost) [reply packages/components/psammead-story-promo/src/index.jsx]: I had already created an issue on this here is the link. https://github.com/bbc/psammead/issues/1921

File: packages/components/psammead-story-promo/src/index.jsx [hunk 2]
```diff
@@ -141,19 +142,15 @@ const InlineMediaIndicator = styled.div`
...
-    return topStory ? getParagon(script) : getPica(script);
-  }}
   color: ${C_EBON};
   ${({ service }) => getSerifMedium(service)}
   margin: 0; /* Reset */
   padding-bottom: ${GEL_SPACING};
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    ${props => (props.script ? getParagon(props.script) : '')};
+    ${({ script, topStory }) =>
+      script && topStory ? getParagon(script) : getGreatPrimer(script)}
```
Comment (andrew-nowak) [inline packages/components/psammead-story-promo/src/index.jsx]:
```suggestion
      script && (topStory ? getParagon(script) : getGreatPrimer(script))}
```

File: packages/components/psammead-story-promo/src/index.jsx (hunk 1 above)
Comment (DenisHdz) [reply packages/components/psammead-story-promo/src/index.jsx]: @jroebu14 Could you also leave your thoughts on https://github.com/bbc/psammead/issues/1921, please?

File: packages/components/psammead-story-promo/src/index.jsx [hunk 3]
```diff
@@ -141,19 +142,15 @@ const InlineMediaIndicator = styled.div`
 export const Headline = styled.h3`
   ${props => (props.script ? getPica(props.script) : '')};
 
-  ${({ script, topStory }) => {
-    if (!script) {
-      return '';
-    }
+  ${({ script, topStory }) => (script && topStory ? getParagon(script) : '')}}
```
Comment (andrew-nowak) [inline packages/components/psammead-story-promo/src/index.jsx]: This doesn't look right, should this line be merged with the one above?

**Review by andrew-nowak — CHANGES_REQUESTED** [review]

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

File: packages/components/psammead-story-promo/src/index.jsx (hunk 3 above)
Comment (andrew-nowak) [reply packages/components/psammead-story-promo/src/index.jsx]: I think the lines you deleted on the left hand side of this diff were good, just the `${props => (props.script ? getPica(props.script) : '')};` above needed to be deleted

Comment (ghost) [reply packages/components/psammead-story-promo/src/index.jsx]: Applied some changes, but am not sure if i got your suggestion right.

File: packages/components/psammead-story-promo/src/index.jsx [hunk 4]
```diff
@@ -139,21 +140,16 @@ const InlineMediaIndicator = styled.div`
 `;
 
 export const Headline = styled.h3`
-  ${props => (props.script ? getPica(props.script) : '')};
+  ${({ script, topStory }) =>
+    script && topStory ? getParagon(script) : getPica(script)}}
```
Comment (andrew-nowak) [inline packages/components/psammead-story-promo/src/index.jsx]:
```suggestion
    script && (topStory ? getParagon(script) : getPica(script))}
```
I think this is also needed, but definitely looks better :+1:

**Review by andrew-nowak — APPROVED** [review]
yep lgtm

**Review by Bopchy — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**